src/Tooltip.jsx

```diff
@@ -1,9 +1,22 @@
 import React, { Component } from 'react';
+import { findDOMNode } from 'react-dom';
 import PropTypes from 'prop-types';
+import positions from 'positions';
 import Trigger from 'rc-trigger';
 import { placements } from './placements';
 import Content from './Content';
 
+const placementsMap = {
+  tc: 'top center',
+  bc: 'bottom center',
+  cl: 'center right',
+  cr: 'center left',
```

Comment:
Shouldn't value of this one be `center right`, and previous one other way around ?

Reply:
Sure)